src/ResourceManager/Compute/Commands.Compute/Extension/SqlServer/NewAzureRmVMSqlServerAutoBackupConfig.cs

```diff
@@ -0,0 +1,321 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Compute.Common;
+using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Common.Authentication.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common;
+using Microsoft.Azure.Management.Storage;
+using Microsoft.WindowsAzure.Commands.Common.Storage;
+using System;
+using System.Management.Automation;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Security.Permissions;
+
+namespace Microsoft.Azure.Commands.Compute
+{
+    /// <summary>
+    /// Helper cmdlet to construct instance of AutoBackup settings class
+    /// </summary>
+    [Cmdlet(
+        VerbsCommon.New,
+        ProfileNouns.VirtualMachineSqlServerAutoBackupConfig,
+        DefaultParameterSetName = StorageUriParamSetName),
+    OutputType(
+        typeof(AutoBackupSettings))]
+    public class NewAzureRmVMSqlServerAutoBackupConfigCommand : AzureRMCmdlet
```

Comment:
@anilyela if you are changing the name of the cmdlet `New-AzureVMSqlServerAutoBackupConfig` to `New-AzureRmVMSqlServerAutoBackupConfig`, you do not need to add an entirely new file - you can add an alias in the old cmdlet to `New-AzureRmVMSqlServerAutoBackupConfig`. There does not need to be separate files for the same cmdlet.